intro-section.tex

```diff
@@ -75,9 +75,12 @@ \subsection{Hello World in cylc}
 \terminology{A cylc \underline{suite daemon} is a light-weight server program
 dedicated to managing a single workflow.}
 
-\paragraph*{Demo}
-Copy the Hello World example code into a suite.rc file in a directory called
-hello\_world, and run the following commands:
+\begin{shaded*}
+
+\subsection{Hello World Tutorial}
+
+Copy the Hello World example code above into a {\em suite.rc} file in a directory called
+{\em hello\_world}, and run the following commands:
 
 \begin{lstlisting}[language=bash]
```

Comment:
The default code colour is quite similar to the shaded background making it a little difficult to read listings.